src/material.js

```diff
@@ -77,10 +77,15 @@ export class Material {
             basis: this.Basis.toJSON(),
             name: this.name || this.formula,
             isNonPeriodic: this.isNonPeriodic || false,
+            hash: this.hash,
         };
     }
 
     clone(extraContext) {
+        const resetHash = extraContext ? ("resetHash" in extraContext) : false;
+        if (resetHash) {
+            this.setProp('hash', '');
```

Comment:
This will reset the has for the current material too, we don't want that.

Moreover, `extraContext` already allows us to reset hash by just passing: `{hash: ""}` when invoking. Nothing else.